museek/plugin/sanity_check_observation_plugin.py

```diff
@@ -220,3 +222,56 @@ def create_plots_of_scan_data(self, data: TimeOrderedData):
         plt.hist(data.elevation.squeeze.flatten(), bins=200)
         plt.xlabel('elevation')
         self.savefig(description='Elevation histogram of all dishes during scan.')
+
```

Comment:
ok

Reply:
How to use autopep8?  I change the settings and make it do the format automatically .

Reply:
you can install it as an extension and then set the max line length to 120

Reply:
I installed it, which parameter to set ?

Reply:
in vs code you can use ctrl-shift-p to access some kind of command prompt. In there, type "user settings" and click on the option that ends with "(JSON)". It will open the json file that controls all of vs code.
In there you need to add
``` 
{   
    "[python]": {
        "editor.formatOnType": true,
        "editor.defaultFormatter": "ms-python.autopep8"
    },
    "autopep8.args": [
        "--max-line-length=120"
    ],
}
    ```

Reply:
you can also do it via the gui of vs code...